### PR TITLE
Placeholdes display

### PR DESCRIPTION
**Pull request by moridyn**

### CODE REVIEW

File: djangocms_rest_api/serializers/base.py [hunk 1]
```diff
@@ -239,12 +239,14 @@ def _get_declared_fields(attrs):
...
-def get_serializer_class(plugin=None, model=None):
+def get_serializer_class(plugin=None, model=None, instance=None):
     serializer_class = None
     if plugin:
+        # import pdb
+        # pdb.set_trace()
         serializer_class = getattr(plugin, 'serializer_class', None)
-        if not serializer_class:
-            serializer_class = plugin_serializer_mapping.get(plugin.plugin_type)
+        if not serializer_class and instance:
```
Comment (czpython) [inline djangocms_rest_api/serializers/base.py]: why pass instance? `plugin.plugin_type` and `instance.plugin_type` should always match

**Review by czpython — CHANGES_REQUESTED** [review]

File: djangocms_rest_api/serializers/base.py (hunk 1 above)
Comment (moridyn) [reply djangocms_rest_api/serializers/base.py]: plugin does not have attribute plugin_type or at least some like LinkPlugin. Got AttributeError in there

Comment (czpython) [reply djangocms_rest_api/serializers/base.py]: It has to have it.. `LinkPlugin` inherits from `CMSPlugin` which has it.
If it throws an `AttributeError` then its not good behavior, please have a look.

File: djangocms_rest_api/serializers/base.py [hunk 2]
```diff
@@ -112,11 +112,11 @@ class Meta:
 
     def get_plugin_data(self, obj):
 
-        instance, plugin = obj.get_plugin_instance()
+        plugin = obj.get_plugin_class()
         model = getattr(plugin, 'model', None)
         if model:
             serializer = get_serializer(
-                instance, model=getattr(plugin, 'model', None), plugin=plugin, context=self.context)
+                obj, model=getattr(plugin, 'model', None), plugin=plugin, context=self.context)
```
Comment (czpython) [inline djangocms_rest_api/serializers/base.py]: plugin will always have a `model` attribute.
So you can do `plugin.model`

**Review by czpython — APPROVED** [review]

**Pull request merged**